Lib/multiprocessing/process.py

```diff
@@ -337,6 +348,36 @@ def __reduce__(self):
                 )
         return AuthenticationString, (bytes(self),)
 
+
+#
+# Create object representing the parent process
+#
+
+class _ParentProcess(BaseProcess):
+
+    def __init__(self, name, pid, sentinel):
+        self._identity = ()
+        self._name = name
+        self._pid = pid
+        self._parent_pid = None
+        self._popen = None
+        self._closed = False
+        self._sentinel = sentinel
+        self._config = {}
+
+    def close(self):
+        pass
+
+    def is_alive(self):
```

Comment:
Should `join()` be overriden as well? Or does it work already? I don't see it in the tests.